tests/test_ftp_parse.py

```diff
@@ -164,39 +165,50 @@ def test_decode_linux(self, mock_localtime):
     def test_decode_windowsnt(self, mock_localtime):
         mock_localtime.return_value = time2017
         directory = """\
+unparsable line
 11-02-17  02:00AM       <DIR>          docs
 11-02-17  02:12PM       <DIR>          images
-11-02-17  02:12PM       <DIR>          AM to PM
+11-02-17 02:12PM <DIR> AM to PM
 11-02-17  03:33PM                 9276 logo.gif
+05-11-20   22:11  <DIR>       src
+11-02-17   01:23       1      12
+11-02-17    4:54                 0 icon.bmp
```

Comment:
Just for the sake of completeness, might be worth also adding e.g.
```
11-02-17    4:54AM                 0 icon.gif
11-02-17    4:54PM                 0 icon.png
```
? :shrug:

Reply:
Done